1-exercises/A-accessing-values/exercise3.js

```diff
@@ -20,7 +20,11 @@ let basketballTeam = {
     - console.logs the name of each player on a new line
```

Comment:
I think I don't see the console.log in your code. Maybe you can call the function, and output the string it returns?